File: 프로그래머스/1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.cs
public class Solution {
    public string solution(string s) {
        int len = s.Length;
        if(len % 2 == 1)
        {
            return s[len / 2].ToString();
        }
        else
        {
            return s[len / 2 - 1].ToString() + s[len / 2].ToString();
        }
    }
}

File: 프로그래머스/1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string s) {
    int n = s.size();
    string ans = "";
    if(n % 2 == 1)
    {
        return ans += s[n / 2];
    }
    else
    {
        ans += s[n / 2 - 1];
        ans += s[n / 2];        
        return ans;
    }
}